registry/tracecat_registry/templates/tools/wazuh/active_response.yml

```diff
@@ -0,0 +1,48 @@
+type: action
+definition:
+  name: wazuh_active_response
+  namespace: tools.wazuh
+  title: Wazuh Agents Active Response
+  description: Run an Active Response command on all agents or a list of them
+  display_group: Wazuh
+  doc_url: null
+  author: null
+  expects:
+    command:
+      type: str
+      description: Command running in the agent. If this value starts with !, then it refers to a script name instead of a command name
```

Comment:
```suggestion
      description: Command running in the agent. If this value starts with !, then it refers to a script name instead of a command name.
```